### DISCUSSION TITLE
Email to contact person with list of attendees

### DISCUSSION THREAD

**Discussion by WXQ8NPb5TrbDNqzywcxF** [liedekef/events-made-easy#354] [General]
Hi !,
Hope my question is not stupid :|
Is there a way to send en email, the morning of an event, containing the list of all attendees ?
I'm trying to avoid to ask the contact person to log within the admin interface.

Thanks for your great product !

**Comment by liedekef** [comment]
Sorry, that is not possible. But you can create a list of attendees using [eme_attendees] (or [eme_bookings] and hide that behind a page with a password on it or so, or if the person is logged in, use [eme_if] surrounding [eme_attendees] and as condition check if the person is logged in.

**Reply by WXQ8NPb5TrbDNqzywcxF** [reply]
Thanks !  I think i've been able to make it with hook and actions.  thanks !
